cmip6_downscaling/data/cmip.py

```diff
@@ -93,64 +87,57 @@ def load_cmip(
     """
     with dask.config.set(**{'array.slicing.split_large_chunks': False}):
 
-        if isinstance(variable_ids, str):
-            variable_ids = [variable_ids]
-
         col = cat.cmip6()
+        col_subset = col.search(
+            activity_id=activity_ids,
+            experiment_id=experiment_ids,
+            member_id=member_ids,
+            source_id=source_ids,
+            table_id=table_ids,
+            grid_label=grid_labels,
+            variable_id=variable_ids,
+        )
+        if len(col_subset.keys()) > 1:
+            raise ValueError('More than one search result exist in intak-esm search')
 
-        for i, var in enumerate(variable_ids):
-            stores = (
-                col.search(
-                    activity_id=activity_ids,
-                    experiment_id=experiment_ids,
-                    member_id=member_ids,
-                    source_id=source_ids,
-                    table_id=table_ids,
-                    grid_label=grid_labels,
-                    variable_id=[var],
-                )
-                .df['zstore']
-                .to_list()
-            )
-
-            # Q: why are we using the era5 storage_options
-            storage_options = config.get('data_catalog.era5.storage_options')
-            if len(stores) > 1:
-                raise ValueError('can only get 1 store at a time')
-
-            ds = xr.open_zarr(stores[0], storage_options=storage_options).pipe(postprocess)
+        ds = col_subset[list(col_subset.keys())[0]].to_dask().pipe(postprocess)
 
-            # convert to mm/day - helpful to prevent rounding errors from very tiny numbers
-            if var == 'pr':
-                ds['pr'] *= 86400
+        # convert to mm/day - helpful to prevent rounding errors from very tiny numbers
+        if variable_ids == 'pr':
+            ds['pr'] *= 86400
```

Comment:
Another (better) option here is to check if `pr` is in `ds`:
```suggestion
        if 'pr' in ds:
            ds['pr'] *= 86400
```

Related to my comment above, this will handle the case where we get a dataset with multiple variables in it.